app/views/shared/check_answers/_only_link_section.html.erb

```diff
@@ -2,10 +2,11 @@
   <div class="govuk-grid-column-two-thirds">
     <h2 class="govuk-heading-m"><%= question %></h2>
   </div>
+  <%= link_to(url, class: 'govuk-link change-link') do %>
     <div class="govuk-grid-column-one-third govuk-summary-list--no-border align-text-right">
```

Comment:
I'm surprised webhint is not complaining about a `div` being inside a

Reply:
I just looked this up:
> HTML 5 states that the <a> element "may be wrapped around entire paragraphs, lists, tables, and so forth, even entire sections, so long as there is no interactive content within (e.g. buttons or other links)".

Webhint is complaining about `legend`s without `fieldset`s, but I shall fix that separately - this ticket is already big enough.

Reply:
Thanks for looking up the info 👍